sdks/python/apache_beam/runners/dataflow/dataflow_runner_test.py

```diff
@@ -258,6 +259,18 @@ def test_biqquery_read_streaming_fail(self):
                     PipelineOptions(self.default_properties)) as p:
         _ = p | beam.io.Read(beam.io.BigQuerySource('some.table'))
 
+  def test_biqquery_read_uw_fail(self):
```

Comment:
fnapi_fail (to be consistent with the above check).

Reply:
Done.